File: SDKs/probely.com/1.2.0/go/pkg/models/operations/deletewebhooksid.go
package operations

type DeleteWebhooksIDPathParams struct {
	ID string `pathParam:"style=simple,explode=false,name=id"`
}

type DeleteWebhooksID401ApplicationJSON struct {
	Detail *string `json:"detail,omitempty"`
}

type DeleteWebhooksID403ApplicationJSON struct {
	Detail *string `json:"detail,omitempty"`
}

type DeleteWebhooksID404ApplicationJSON struct {
	Detail *string `json:"detail,omitempty"`
}

type DeleteWebhooksIDRequest struct {
	PathParams DeleteWebhooksIDPathParams
}

type DeleteWebhooksIDResponse struct {
	ContentType                              string
	StatusCode                               int64
	DeleteWebhooksID401ApplicationJSONObject *DeleteWebhooksID401ApplicationJSON
	DeleteWebhooksID403ApplicationJSONObject *DeleteWebhooksID403ApplicationJSON
	DeleteWebhooksID404ApplicationJSONObject *DeleteWebhooksID404ApplicationJSON
}

File: SDKs/up.com.au/v1/python/src/sdk/models/operations/delete_webhooks_id_.py
import dataclasses



@dataclasses.dataclass
class DeleteWebhooksIDPathParams:
    id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'id', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class DeleteWebhooksIDRequest:
    path_params: DeleteWebhooksIDPathParams = dataclasses.field()
    

@dataclasses.dataclass
class DeleteWebhooksIDResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    
